src/operators.jl

```diff
@@ -397,7 +397,7 @@ end
 ###############################################################################
 
 const _OP_HINT = "Are you trying to build a nonlinear problem? Make sure you use @NLconstraint/@NLobjective."
-for (func, _) in Calculus.symbolic_derivatives_1arg(),
+for func in MOI.Nonlinear.DEFAULT_UNIVARIATE_OPERATORS[4:end],
```

Comment:
I would avoid using a magic number here (why discarding the three first terms in the list?). How about implementing a proper getter in the MOI side to return the [following operators](https://github.com/jump-dev/MathOptInterface.jl/blob/master/src/Nonlinear/univariate_expressions.jl#L15-L83)?

Reply:
I've refactored this loop. We ignored `+`, `-`, and `abs` because they were defined separately in `_Derivatives` and so weren't included here. Skipping `+` and `-` makes sense because they're implemented separately, but I don't know why `abs` was skipped.